.github/workflows/maven.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Maven Central Repository
+        uses: actions/setup-java@v1
+        with:
+          java-version: '11'
```

Comment:
I'm surprised this works without the `distribution:` tag here. The [setup-java docs](https://github.com/actions/setup-java#usage) say it's required and I've had other builds fail because it wasn't included.

We're using `distribution: 'adopt'` in the gtfs-validator and gtfs-realtime-validator, so I'd suggest adding the same here so we don't have problems with it failing later.

Reply:
Well, I just noticed this in https://github.com/actions/setup-java#supported-distributions:

>NOTE: Adopt OpenJDK got moved to Eclipse Temurin and won't be updated anymore. It is highly recommended to migrate workflows from adopt to temurin to keep receiving software and security updates. See more details in the [Good-bye AdoptOpenJDK post](https://blog.adoptopenjdk.net/2021/08/goodbye-adoptopenjdk-hello-adoptium/).

So we should instead use `distribution: 'temurin'` here, and change the gtfs-validator and gtfs-realtime-validator to do the same. I'll open PRs for those projects.

Reply:
I modified it and the updated workflow is working fine. Thank you for noticing me!